ui-participant/src/hub/HubPage.tsx

```diff
@@ -18,6 +18,16 @@ export default function HubPage() {
   const { portal, portalEnv } = usePortalEnv()
   const { enrollees } = useUser()
   const [noActivitiesAlert, setNoActivitiesAlert] = useState<ParticipantDashboardAlert>()
+  const [activeEnrolleeId, setActiveEnrolleeId] = useState(enrollees[0]?.id)
+
+  // Function to change the active enrollee
+  const handleTabClick = (enrolleeId: string)  => {
+    setActiveEnrolleeId(enrolleeId)
+  }
+
+  // Find the active enrollee based on activeEnrolleeId
+  const activeEnrollee = enrollees.find(enrollee => enrollee.id === activeEnrolleeId)
+
 
```

Comment:
This should most probably moved to another branch, I was using the data to come up with a design for the dashboard for a proxy. My initial design was to use tabs and to use notification alerts as a signal for a task waiting. I don't have approval from Erin so I will move these changes to another local branch and revert here, but I thought since it is already here I might as well attach a screenshot here too
![Screenshot 2024-02-08 at 1 29 59 PM](https://github.com/broadinstitute/juniper/assets/47120870/13baa6e9-68a4-4507-a86a-7fd4d17eecb8)

Reply:
looks reasonable, and yes, please move to another branch :)